docs/ctf-bots.md

```diff
@@ -0,0 +1,35 @@
+# CTF Q-bots
+
+Source code of bots was miss, for now we only have [binaries](https://github.com/airmash-refugees/Q-bots)
+
+## Commands
+
+### Common
+
+* `#status` - show amount of bots in your team
+* `#help` - show help for bots commands
+* `#help <command without #>` to see more details about specified command
+
+### Leadership
+
+Bots choose leader inside a team, who can give commands to bot.
+
+* If current leader is AFK you can start elections with:
+    * send `#status` command
+    * in next 10 seconds send `/elections` command
+
+* If you are leader and want to pass leadership to another player: `#leader <NICKNAME>`
+
+### Bot commands
+
+* `#cap` (or `#capture`, `#escort`) - bots will help to capture flag and escort player with the flag to your base
+* `#recap` (or `#recover`) - bots will try to return stolen flag
+* `#defend` - bots will defend your base
+* `#auto` - auto select strategy between cap|recap|defend
+* `#assist <player | me>` - bots will assist to specified player, if `me` typed bots will assist you
+* `#drop` - bot, which currently have a flag will drop it to another player
```

Comment:
In addition to `#drop`, there is a [recently added](https://github.com/wight-airmash/ab-server/blob/master/CHANGES.md#580-february-28-2020) `#dropnow` custom command.

Reply:
Updated